doc/api/stream.md

```diff
@@ -230,6 +230,21 @@ that no more events will be emitted, and no further computation will occur.
 A [`Writable`][] stream will always emit the `'close'` event if it is
 created with the `emitClose` option.
 
+##### Event: 'destroy'
+<!-- YAML
+added: REPLACEME
+changes:
+  - version: v10.0.0
+    pr-url: https://github.com/nodejs/node/pull/29791
```

Comment:
If this is new the `changes` isn't necessary. Otherwise `v10.0.0` is incorrect and `description` is missing.

Reply:
Thankss!